.github/workflows/notebook-test-cron.yml

```
@@ -29,7 +29,19 @@ jobs:
           instance: "client-enablement/documentation/qiskit-documenta"
 
       - name: Execute notebooks
-        run: tox -- --only-submit-jobs
+        timeout-minutes: 330
```

This is 5hr 30mins, which is 30mins short of the maximum 6hrs. This means if jobs take too long out we'll still have some time to finish the action and upload any notebooks that did execute.

---

Upload should be quite fast. I recommend 350, along with a comment explaining thew workflow max is 360 minutes so we leave 10 minutes for the other steps. Or 345 works, but I doubt the other steps take 15 minutes.

---

That's true. Looking at the job, the other steps take ~1m total. I'll still leave 10mins just in case.

```suggestion
        timeout-minutes: 350
```